docker/compose/mozdef_meteor/Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN mkdir -p /opt/mozdef/envs/meteor/mozdef
 RUN if [ "${METEOR_BUILD}" = "YES" ]; then \
         cd /opt/mozdef/envs/mozdef/meteor && \
         meteor npm install && \
-        meteor build --server localhost:3002 --directory /opt/mozdef/envs/meteor/mozdef && \
+        echo "Starting meteor build" && \
+        time meteor build --server localhost:3002 --directory /opt/mozdef/envs/meteor/mozdef && \
```

Comment:
Do we still want to include this time command? Or was this for debugging purposes?

Reply:
I'd hoped to include it as it's useful information during the build to show what just happened for the last 3 minutes. I looked into how to get `meteor build` to output some kind of progress our output and had no luck. So yes, I think we should keep the `time` in. Is there any downside?

Reply:
I'm fine with keeping it in, just curious if it was intentional or not.